lib/events/stream.go

```diff
@@ -161,6 +161,13 @@ func (s *ProtoStreamer) ResumeAuditStream(ctx context.Context, sid session.ID, u
 	})
 }
 
+// GetUploadMetadata gets the session upload metadata
+// Implements events.UploadMetadataGetter
+func (s *ProtoStreamer) GetUploadMetadata(sid session.ID) *UploadMetadata {
+	uploadMetadata := s.cfg.Uploader.GetUploadMetadata(sid)
```

Comment:
You can avoid the local variable if you changed to return by value (like MultipartUploader).